README.md

```diff
@@ -8,55 +8,90 @@
 Stencil Playwright is a testing adapter package that allows developers to easily use the [Playwright testing framework](https://playwright.dev/docs/intro)
 in their Stencil project.
 
-For full documentation, please see the [Playwright testing docs on the official Stencil site](https://stenciljs.com/docs/next/testing/playwright/overview).
+For full documentation, please see the [Playwright testing docs on the official Stencil site](https://stenciljs.com/docs/testing/playwright/overview).
 
 ## Getting Started
 
-1. Install the necessary packages using your preferred package manager:
+1. Install the necessary dependencies:
 
-   ```bash
+   ```bash npm2yarn
    npm i @stencil/playwright @playwright/test --save-dev
    ```
 
-1. Install Playwright's browser binaries:
+1. Install the Playwright browser binaries:
 
    ```bash
    npx playwright install
    ```
 
-1. Create a `playwright.config.ts` file. The `@stencil/playwright` package offers a utility function to quickly create a default configuration that should
-   work for most Stencil projects:
+1. Create a Playwright config at the root of your Stencil project:
 
    ```ts
-   // playwright.config.ts
-
    import { expect } from '@playwright/test';
    import { matchers, createConfig } from '@stencil/playwright';
 
+   // Add custom Stencil matchers to Playwright assertions
    expect.extend(matchers);
 
-   export default createConfig();
+   export default createConfig({
+     // Overwrite Playwright config options here
+   });
    ```
 
-1. At this point, any tests can be executed using `npx playwright test` (or by updating the project's test command in the `package.json`). By default, Playwright's test matcher
-   is configured to run all tests matching the pattern `*.e2e.ts`. This can be changed by overriding the default matcher using the
-   [`createConfig()` function](#createconfigoverrides-createstencilplaywrightconfigoptions-promiseplaywrighttestconfig):
+   The `createConfig()` is a utility that will create a default Playwright configuration based on your project's Stencil config. Read
```

Comment:
"The _____ is a..." reads a little funky to me - thoughts on:

```suggestion
  `createConfig()` is a utility that will create a default Playwright configuration based on your project's Stencil config. Read
```
or
"The `createConfig()` function is a utility"